### PR TITLE
Run tests with testing environment

### PR DESCRIPTION
**Pull request by MarekChoinski**

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4358 Run tests with testing environment](https://app.clickup.com/t/2549495/SALEOR-4358)

File: recordings/auth-api_3443157535/changes-user-s-password_3225638244/recording.har [hunk 1]
```diff
@@ -47,10 +47,10 @@
             },
             {
               "name": "host",
-              "value": "master.staging.saleor.cloud"
+              "value": "jwt-expire-test.api.saleor.rocks"
```
Comment (dominik-zeglen) [inline recordings/auth-api_3443157535/changes-user-s-password_3225638244/recording.har:50]: Shouldn't we omit hostname and other instance-specific headers?
Or maybe whitelist only those meaningful?

Comment (jwm0) [reply recordings/auth-api_3443157535/changes-user-s-password_3225638244/recording.har:50]: These are not used for matching, and will not be regenerated when the test remains unchanged. I'm neutral on this change, I don't see this information as something that needs obfuscation.

**Review by jwm0 — APPROVED** [review]

File: recordings/auth-api_3443157535/changes-user-s-password_3225638244/recording.har (hunk 1 above)
Comment (dominik-zeglen) [reply recordings/auth-api_3443157535/changes-user-s-password_3225638244/recording.har:50]: I know they are not used, but it generates bigger diffs. Out of scope of this PR though.

**Pull request merged**